### DISCUSSION TITLE
Is mujoco differentiable?

### DISCUSSION THREAD

**Discussion by yzd0014** [google-deepmind/mujoco#1073] [General]
I'm wondering if there are functions in mujoco to compute for example, derivatives of rigid bodies' states with respect to actuators' control.

**Comment by kevinzakka** [comment]
See the section on [derivatives](https://mujoco.readthedocs.io/en/latest/computation/index.html#derivatives).

**Reply by yzd0014** [reply]
Thank you very much!
